addons/website/views/website_navbar_templates.xml

```diff
@@ -60,6 +60,14 @@
                             <a role="menuitem" data-action="promote-current-page" href="#" title="Promote page on the web" class="dropdown-item">Optimize SEO</a>
                         </div>
                     </li>
+                    <li groups="website.group_website_publisher" t-if="'website_published' in main_object.fields_get() and ('can_publish' not in main_object.fields_get() or main_object.can_publish)" t-attf-class="js_publish_management d-block d-sm-block d-md-none #{main_object.website_published and 'css_published' or 'css_unpublished'}" t-att-data-id="main_object.id" t-att-data-object="main_object._name" t-att-data-controller="publish_controller">
```

Comment:
add missing data-display="static"on previous A to push this under the dropdown

Reply:
Can you try to make this part more generic instead to copy/past all the button ?
(t-call ?)